system_metrics_collector/src/system_metrics_collector/utilities.cpp

```diff
@@ -63,7 +63,7 @@ ProcCpuData processStatCpuLine(const std::string & stat_cpu_line)
   ProcCpuData parsed_data;
 
   if (!stat_cpu_line.empty()) {
-    if (!stat_cpu_line.compare(0, strlen(CPU_LABEL), CPU_LABEL)) {
+    if (!stat_cpu_line.compare(0, strlen(kCpuLabel), kCpuLabel)) {
```

Comment:
In another PR, can you insert a comment here as to what this line is doing?

Reply:
Sure!